x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -74,6 +74,8 @@
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_LOADING_DATA = "Started loading data";
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_ANALYZING = "Started analyzing";
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_WRITING_RESULTS = "Started writing results";
+    public static final String DATA_FRAME_ANALYTICS_CANNOT_UPDATE_IN_CURRENT_STATE =
+        "Cannot update analytics [{0}] while its status is {1}";
```

Comment:
change `status` to `state`

Reply:
I also wonder here if it'd be more informative to say that we cannot update the job unless its in the stopped state. What do you think

Reply:
Also, if we keep as is, we should wrap `{1}`in square brackets.

Reply:
> I also wonder here if it'd be more informative to say that we cannot update the job unless its in the stopped state. What do you think

SGTM